src/drivers/imx/sai.c

```diff
@@ -301,7 +303,7 @@ static inline int sai_set_config(struct dai *dai, struct ipc_config_dai *common_
 		dai_info(dai, "SAI: codec is consumer");
 		val_cr2 |= REG_SAI_CR2_MSEL_MCLK1;
 		val_cr2 |= REG_SAI_CR2_BCD_MSTR;
-		val_cr2 |= SAI_CLOCK_DIV; /* TODO: determine dynamically.*/
+		val_cr2 |= clk_div; /* TODO: determine dynamically.*/
```

Comment:
Is this comment still valid? Ditto for a few other instances inside this file.

Reply:
Yes. Other platforms still use a fixed value. This has not changed.

Reply:
Hmm okay, fair enough then.